test/testutil.py

```diff
@@ -1,36 +1,20 @@
-import functools
-import logging
 import operator
 import os
-import random
 import socket
-import string
 import time
 import uuid
 
-from six.moves import xrange
```

Comment:
@jeffwidman Yes, I missed this. I'll take care of this in the upcoming PRs, if you don't mind.

Reply:
No worries, at all, I'm happy to clean it up myself after merging